sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_metrics_v1/query.sql

```diff
@@ -1,477 +1,225 @@
+  LEFT JOIN UNNEST(metrics.labeled_counter.bookmarks_edit) AS bookmarks_edit_table
+  LEFT JOIN UNNEST(metrics.labeled_counter.bookmarks_open) AS bookmarks_open_table
+  LEFT JOIN UNNEST(metrics.labeled_counter.bookmarks_view_list) AS bookmarks_view_list_table
+  WHERE
+    LOWER(metadata.isp.name) <> "browserstack"
+    -- we need to work with a larger time window as some metrics ping arrive with a multi day delay
+    AND DATE(submission_timestamp) BETWEEN DATE_SUB(@submission_date, INTERVAL 4 DAY) AND @submission_date
+    AND DATE(DATETIME(LEAST(ping_info.parsed_start_time, ping_info.parsed_end_time), 'UTC')) = DATE_SUB(@submission_date, INTERVAL 4 DAY)
+GROUP BY `date`, client_id, channel, country
 )
+-- Aggregated feature usage
 SELECT
-  submission_date,
-  metrics_ping_distinct_client_count,
-/*Logins*/
-  logins_deleted_users,
-  logins_deleted,
-  logins_modified_users,
-  logins_modified,
-  logins_saved_users,
-  logins_saved,
-  logins_saved_all_users,
-  logins_saved_all,
-/*Credit Card*/
-  credit_card_autofill_enabled_users,
-  credit_card_autofill_enabled,
-  credit_card_sync_enabled_users,
-  credit_card_sync_enabled,
-/*Bookmark*/
-  bookmarks_add_users,
-  bookmarks_add,
-  bookmarks_delete_users,
-  bookmarks_delete,
-  bookmarks_edit_users,
-  bookmarks_edit,
-  has_mobile_bookmarks_users,
-  has_mobile_bookmarks,
-  mobile_bookmarks_count_users,
-  mobile_bookmarks_count,
-  bookmarks_open_users,
-  bookmarks_open,
-  bookmarks_view_list_users,
-  bookmarks_view_list,
-/*FxA*/
-  sync_create_account_pressed_users,
-  sync_create_account_pressed,
-  sync_open_tab_users,
-  sync_open_tab,
-  sync_sign_in_sync_pressed_users,
-  sync_sign_in_sync_pressed,
-/*Privacy*/
-  tabs_private_tabs_quantity_users,
-  tabs_private_tabs_quantity,
-  preferences_close_private_tabs_users,
-  preferences_close_private_tabs,
-  tracking_protection_enabled_users,
-  tracking_protection_enabled,
-  tracking_protection_strict_users,
-  tracking_protection_strict,
-/*Tab Count*/
-  tabs_normal_tabs_quantity_users,
-  tabs_normal_tabs_quantity,
-  tabs_inactive_tabs_count_users,
-  tabs_inactive_tabs_count,
-/*Default Browser*/
-  app_opened_as_default_browser_users,
-  app_opened_as_default_browser,
-  settings_menu_set_as_default_browser_pressed_users,
-  settings_menu_set_as_default_browser_pressed,
-/*Notification*/
-  preferences_sync_notifs_users,
-  preferences_sync_notifs,
-  preferences_tips_and_features_notifs_users,
-  preferences_tips_and_features_notifs,
-/*Customize Home*/
-  preferences_jump_back_in_users,
-  preferences_jump_back_in,
-  preferences_recently_visited_users,
-  preferences_recently_visited,
-  preferences_recently_saved_users,
-  preferences_recently_saved,
-  preferences_pocket_users,
-  preferences_pocket,
-  app_menu_customize_homepage_users,
-  app_menu_customize_homepage,
-  firefox_home_page_customize_homepage_button_users,
-  firefox_home_page_customize_homepage_button
+  `date` AS metric_date,
```

Comment:
I wonder if perhaps giving this a slightly more generic name (ping type agnostic) would be a better option. For example, `ping_date`. We could have the same name for this field inside the other usage table and more easily combine it with the `USING()` syntax instead of the more verbose `ON`.